.bazelrc

```diff
@@ -283,7 +283,6 @@ build:remote-clang-cl --config=clang-cl
 build:remote-clang-cl --config=rbe-toolchain-clang-cl
 
 # Docker sandbox
-# NOTE: Update this from https://github.com/envoyproxy/envoy-build-tools/blob/main/toolchains/rbe_toolchains_config.bzl#L8
```

Comment:
im wondering why remove this - it seems like that line is the authoritative place to get the image hash from

Reply:
No update for several months, not sure how image upgrade works.